pom.xml

```diff
@@ -642,7 +642,7 @@
         <commons-codec.version>1.10</commons-codec.version>
         <org.wso2.carbon.identity.association.account>5.1.5</org.wso2.carbon.identity.association.account>
         <identity.extension.utils>1.0.8</identity.extension.utils>
-        <carbon.identity.framework.version>5.16.73</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.17.43</carbon.identity.framework.version>
```

Comment:
An upate regarding proxyContext paths to the fetchAbsoluteURL and fetchRelativeURL was done with 5.17.47. But that would not make any difference to the usage here. Wondering whether we should bump the version here just in case

Reply:
Updated to the latest version.